### PR TITLE
`gw-inventory.php`: Added support for tracking "inventory" on currency-formatted Number fields.

### PR DESCRIPTION
**Pull request by spivurno**
## Context

⛑️ Ticket(s): https://secure.helpscout.net/conversation/2220460632/47304/

## Summary

Customer has a form where users can request funding. There is a hard cap of $10,000 available for funding. He wants to prevent the total amount requested across all entries from exceeding this cap. 

This snippet is well equipped to handle this scenario except that it previously did not support currency-formatted Number fields. Now it does. 👨‍🏭

### CODE REVIEW

**Review by claygriffiths — APPROVED** [review]
Noice!

File: gravity-forms/gw-inventory.php [hunk 1]
```diff
@@ -521,5 +525,5 @@ class GWLimitBySum extends GW_Inventory { }
 	'not_enough_stock_message' => 'You ordered %1$s tickets. There are only %2$s tickets left.',
 	'approved_payments_only'   => false,
 	'hide_form'                => false,
-	'enable_notifications'     => true,
+	'enable_notifications'     => false,
```
Comment (claygriffiths) [inline gravity-forms/gw-inventory.php:528]: Curious, what's the reason for this change?

Comment (spivurno) [reply gravity-forms/gw-inventory.php:528]: Using this option requires the GW_Notificaiton_Event class or it generates a doing it wrong notice. It should be disabled by default.
